src/stats/actions.ts

```diff
@@ -233,13 +277,36 @@ function handleActionCompute(state: PlayerActionState, indices: PlayerIndexedTyp
   incrementCount("wallTechCount.success", currentAnimation === State.WALL_TECH);
   incrementCount("wallTechCount.fail", currentAnimation === State.MISSED_WALL_TECH);
 
-  if (isAerialAttack(currentAnimation)) {
+  if (isAerialLanding(currentAnimation)) {
     incrementCount("lCancelCount.success", playerFrame.lCancelStatus === 1);
     incrementCount("lCancelCount.fail", playerFrame.lCancelStatus === 2);
   }
 
+  // check for edge canceled aerial
+  if (
+    isAerialLanding(prevAnimation) &&
+    (currentAnimation === State.FALL || currentAnimation === State.TEETER) &&
+    playerFrame.internalCharacterId !== null
+  ) {
+    const landingFrames = getPrevAnimationLength(state);
+    const [landingLag, lCanceledLag] = getLandLagFromLandAnimation(prevAnimation, playerFrame.internalCharacterId)!;
+
+    // only counts as a successful edge cancel if an L cancel wouldn't have been faster
+    incrementCount("edgeCancelCount.success", landingFrames < lCanceledLag);
+    incrementCount("edgeCancelCount.slow", landingFrames >= lCanceledLag && landingFrames < landingLag);
+
+    // make edge cancels not count as failed L cancels
+    if (landingFrames <= lCanceledLag && state.lastLCancelStatus === 2) {
+      state.playerCounts.lCancelCount.fail -= 1;
+    }
+  }
+
   // Handles wavedash detection (and waveland)
   handleActionWavedash(state.playerCounts, state.animations);
+
+  if (playerFrame.lCancelStatus && playerFrame.lCancelStatus > 0) {
```

Comment:
I dont think it really matters that much but I assume this first check is supposed to be `exists` as well?